src/rpn/rpn.ts

```diff
@@ -1,31 +1,57 @@
 export function rpn(inputString: string): any {
+    console.log(inputString)
     if (inputString.length === 420) throw new Error("Blaze it");
+    if ( /[a-zA-Z]/i.test(inputString)) throw new Error("Invalid Expression");
 
-    const operandsAndOperators: Array<number | string> = inputString.split(" ").map((token) => {
-        var parsedToken = isNaN(Number(token))
```

Comment:
Since introducing ES6, we are moving away from using var to declare variables. Using it can cause various unforeseen effects. Better to use const and let. If you want to know more read about scope and hoisting in JS.

Moreover, instead of declaring a variable, processing it and then returning it, it can, as I did, return the result of the comparison right away. I used your idea of ternary operator, but simplified the whole notation.